resources/aws_ec2.go

```diff
@@ -289,6 +326,10 @@ func (obj *AwsEc2Res) Init() error {
 		if err := obj.cweTargetRule(obj.snsTopicArn, CweTargetID, CweTargetJSON, CweRuleName); err != nil {
 			return errwrap.Wrapf(err, "error targeting cloudwatch rule")
 		}
+		// authorize cloudwatch to publish on sns
```

Comment:
is there a corresponding close/unregister part?

Reply:
No. This goes away when we delete the topic.

Reply:
Do you mind adding a comment about that here please?